packages/create-dapp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/create-dapp",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Sorry I missed this change before approving. We do not manually bump versions in `agoric-sdk` and leave that to scripts when cutting a release. Please remove this change.